ignite/pkg/relayer/relayer.go

```diff
@@ -200,12 +203,25 @@ func (r Relayer) prepare(ctx context.Context, conf relayerconf.Config, chainID s
 		}
 	}
 
-	key, err := r.ca.ExportHex(chain.Account, "")
+	// Get the key in ASCII armored format
+	passphrase := ""
```

Comment:
@jeronimoalbi So the passphrase is not required ?

Reply:
@tbruyelle good question. To be honest I am not sure about the reason why we were using an empty passphrase.

These changes just make it explicit that we are using an empty passphrase which should be the same for the `Export` and `UnarmorDecryptPrivKey` calls.

Reply:
ah yes I didn't notice it was already the case before! OK for me.